backend/index.js

```diff
@@ -9,6 +9,7 @@ const app = express()
 const port = 8080
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
+app.use('/', questionRouter);
```

Comment:
This is already added further down so I don't think we need this https://github.com/ubclaunchpad/labby/blob/da56a7f3a068f8a10ba52b830c8411cb2192eb01/backend/index.js#L34